### ISSUE TITLE
How to get raw_logits from the Moondream model?

### ISSUE THREAD

**Issue by beingdutta** [m87-labs/moondream#259]
We have the method "model.batch_answer( batch )", but what if I just want to get the logits before the decoding part. Something like using only the forward method "logits = model(batch)" which works for other similar models.
